### PR TITLE
[WIP] XML source support for migration.

### PR DESCRIPTION
**Pull request by joshirohit100**

### CODE REVIEW

File: src/Plugin/DataSource/Xml.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+   */
+  public function process(string $data, array $config) {
+    // Load XML object.
+    $xml = simplexml_load_string($data);
+    $xpath = $item_path = $config['source']['xml_item_selector'];;
+    $simple_xml_elements = $xml->xpath($xpath);
+
+    foreach ($simple_xml_elements as $simple_xml_element) {
+      foreach ($simple_xml_element->children() as $children) {
+        $name = $children->getName();
```
Comment (amitgoyal) [inline src/Plugin/DataSource/Xml.php]: Is this right? aren't we overwriting $name var?

Comment (joshirohit100) [reply src/Plugin/DataSource/Xml.php]: sir thats just kind of pseudo code (for the refernce).  Its not complete but give some idea of how to proceed

Comment (amitgoyal) [reply src/Plugin/DataSource/Xml.php]: hmm.. ok.

**Comment by joshirohit100** [comment]
closing this as @pranitjha raised new PR https://github.com/joshirohit100/migration_glue/pull/3

**Pull request closed without merging**